_posts/2021-11-16-mdacli.md

```diff
@@ -0,0 +1,125 @@
+provided [testfiles][online].
+`topol.tpr` is a GROMACS topolgy file and `traj.trr`
+the corresponding trajectory. The oxygen-oxygen
+RDF can be calculated using
+
+    mda interrdf -s topol.tpr -f traj.trr -g1 "name OW" -g2 "name OW"
+
+The oxygens are selected with the `-g1` and `-g2` flags.
+More verbose output is achieved by using the `-v` flag. Even more,
+information is provided with the `--debug` flag.
+All warnings of the run can also be directly stored to a 
+log file using the `--logfile` flag
+
+    mda --debug --logfile rdf.log interrdf -v -s topol.tpr -f traj.trr -g1 "name OW" -g2 "name OW"
+
+The results of each analysis are stored by default in the current directory.
+The output directory can be changed with the `-o` flag and an additional prefix can be
+set using the `-pre` flag.
+
+The results of the RDF calculations
+are two `.csv` files. The actual RDF is saved in the 2nd and 3rd columns
+of `InterRDF_count_bins_rdf.csv`. The header rows of each file provide
+information about the stored data. Simple results such as bare numbers or
+strings are stored as `JSON` dumps. More complex data such as
+4 or higher-dimensional arrays are saved as a bunch of CSV files zipped
+together.
+
+## Contributions
+
+Contributions are always welcome on [Github][github]. Not only in the form of
+code contributions via pull requests but also issues, especially
+since the project is in an early development phase.
+
+## Motivation and Background
+
+The CLI was developed since
+we were facing the same problem in the labs regularly.
+New students/scientists start writing their analysis
+scripts and face the same challenges and problems i.e.
+
+* How to initialize the universe and loop through frames without copying many lines of code?
+* How to write a CLI parser to analyze several of their simulations?
+* How to process and save their trajectories in a clever way?
+* ...
+
+Some of these problems can be solved by using the
+`MDAnalysis.analysis.base.AnalysisBase` [class of MDA][mda_analysis]. However,
+this class is limited to Python.
+A generic CLI wrapper for all classes based on the AnalysisBase could
+therefore help people to analyze their simulation
+data with the least effort. With this approach, it is easier to use for
+MDA-users since they just stay within their known universe
+with known selection commands and results structures.
+An existing framework makes it also more attractive for users
+and developers to write their analysis using the
+`base.AnalysisBase`. For more technical details visit the
+[documentation][philosophy].
+
+-- @PicoCentauri and @joaomcteixeira
+
+
+[taurenmd]: https://github.com/joaomcteixeira/taurenmd
+[maicos]: https://gitlab.com/maicos-devel/maicos
+[gmx_cli]: https://manual.gromacs.org/documentation/current/user-guide/cmdline.html
+[argparse]: https://docs.python.org/3/library/argparse.html
+[testfiles]: https://github.com/MDAnalysis/mdacli/tree/main/data
+[documentation]: https://mdacli.readthedocs.io/en/latest/
```

Comment:
Change to https://www.mdanalysis.org/mdacli/ before publishing the post?

Reply:
I changed it and we wait with the post until we published the docs on the webpage!